packages/typespec-test/test/batch_modular/generated/typespec-ts/src/rest/responses.ts

```diff
@@ -1320,6 +1320,8 @@ export interface GetTaskFile200Headers {
   "ocp-batch-file-mode": string;
   /** The length of the file. */
   "content-length": number;
+  /** Type of content */
+  "content-type": "application/octet-stream";
```

Comment:
this is a bugfix for previous missing content-type headers https://github.com/microsoft/typespec/issues/3643#issuecomment-2190327538